crates/rust-analyzer/src/config.rs

```diff
@@ -1333,11 +1354,6 @@ impl Config {
         // FIXME @alibektas : Temporary solution. I don't think this is right as at some point we may allow users to specify
         // custom USER_CONFIG_PATHs which may also be relative.
         let user_config_path = VfsPath::from(AbsPathBuf::assert(user_config_path));
```

Comment:
On a follow up note, I feel like we should not make this configurable, where would you even configure this? I guess in the lient config but that still feels a bit odd?

Reply:
by setting a command line argument to the server but ok I will remove it

Reply:
I took a note of it and will make a separate PR for this.